src/python/ensembl/io/genomio/integrity.py

```diff
@@ -313,45 +248,24 @@ def get_gff3(self, gff3_path: Path) -> Dict[str, Dict[str, int]]:
             are stored with their corresponding lengths.
         """
 
-        seqs = {}
-        genes = {}
-        peps = {}
-        all_peps = {}
-        tes = {}
+        seqs: Lengths = {}
+        genes: Lengths = {}
+        peps: Lengths = {}
+        all_peps: Lengths = {}
+        tes: Lengths = {}
```

Comment:
```suggestion
        transposons: Lengths = {}
```
This is a "silly" suggestion because I find `tes` a little bit hard to quickly link with TEs.